### DISCUSSION TITLE
求助，咋买不了了

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#3743] [提问] [answered]
Xbox摆了（梯子开关没用
![屏幕截图 2024-04-27 150755](https://github.com/Hex-Dragon/PCL2/assets/121204926/faa340ac-2537-4ac0-b99f-aa8ad3791026)
![屏幕截图 2024-04-27 150814](https://github.com/Hex-Dragon/PCL2/assets/121204926/8c2ab72e-067d-4da0-949c-b282f5757893)

**Comment by MoYuan-CN** [comment]
我也不太清楚怎么回事，我去询问一下支持团队
Edit: 对话中

**Comment by MoYuan-CN** [comment]
尝试做出如下操作：
1. 修改电脑 DNS 为手动指定：主要 `4.2.2.1` 备用 `4.2.2.2`
2. `Win+R` 输入 inetcpl.cpl，`高级菜单` 中启用 `SSL 3.0` `TLS 1.0` `TLS 1.1` `TLS 1.2`
3. 右键开始徽标，进入管理员模式 CMD，输入 `netsh winsock reset` 重置 winsock
4. 重启电脑

**Reply by WorldHim** [reply]
完成（或仍然失败）后建议将设置改回原设置

**Reply by MoYuan-CN** [reply]
如果还不行的话去找 Xbox 支持
https://support.xbox.com/zh-CN/contact-us
点击 `与我们联系`
选择 `游戏和应用程序` -> `我找不到游戏、DLC、易耗品` -> `电脑`
点击 `跳过`，然后选择 `在线聊天`

**Reply by SqrtTan** [reply]
能用中文聊吗（

**Reply by SqrtTan** [reply]
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/b9610803-13d8-48d1-bc9d-239d41f6f8fd)
666

**Reply by MoYuan-CN** [reply]
66
注意有个支持时间的

> 能用中文聊吗（

可以的

**Reply by wuliaodexiaoluo** [reply]
可以，之前聊过，可以中文

**Comment by allMagicNB** [comment]
其实可以去海鲜市场直接买兑换码，还更便宜

**Reply by SqrtTan** [reply]
其实我也不急，最近跟G胖好上了（

**Comment by Ad-closeNN** [comment]
解决了吗

*~~要不试试把xbox付款页面改为英文试试（从xbox改名知道有些页面要英文才能）~~*
*~~当然这种问题我没见过，只是猜猜罢了，别喷~~*

**Reply by SqrtTan** [reply]
好的，没用（恼

**Reply by allMagicNB** [reply]
那就去海鲜市场买兑换码……

**Comment by SilianZ** [comment]
请在 https://account.microsoft.com/profile/ 将你的“国家或地区”更改为中国（China）。
<img width="375" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/113701655/9f7c7d51-ae4a-4543-b340-c29c4c711d0d">

**Reply by SqrtTan** [reply]
没问题啊

**Comment by Deep-Dark-Forest** [comment]
为什么一定要在官方买呢🤔
不能在其他平台购买兑换码吗

**Reply by Pigeon0v0** [reply]
其他平台兑换码来源说实话不一定可信

**Comment by WTP016-CN** [comment]
尝试使用阿里云DNS？或者使用其他设备试试，如果不行的话查看微软账号年龄设置是否>18

**Comment by Ad-closeNN** [comment]
为什么不重新注册一个新Microsoft账号试试购买（）

**Reply by Deep-Dark-Forest** [reply]
那为什么不直接买个已经购买过Minecraft的账号呢（

**Comment by RimacC2-EV** [comment]
运行网络重置试试？

Windows 设置 - 网络和 Internet - 高级网络设置 - 网络重置 （Windows 11）

**Comment by SqrtTan** [comment]
现在突然能买了！！！

**Reply by SqrtTan** [reply]
~虽然没用上但是~感谢帮助

**Reply by MoYuan-CN** [reply]
啊？

**Reply by SqrtTan** [reply, marked as answer]
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/0a12b3a0-33c3-4608-8d2f-43f1ce6dbc26)
![image](https://github.com/Hex-Dragon/PCL2/assets/121204926/2aa56636-a374-4b2a-9586-eba5403669c2)
